src/web/lib/hakuneko/frontend@classic-dark/menu.html

```diff
@@ -300,6 +310,22 @@
                     this.$.importFile.value = null;
                 }
             }
+
+            minimizeWindow(event){
+                this.win.minimize();
+            }
+
+            maximizeWindow(event){
+                if (!this.win.isMaximized()) {
+                    this.win.maximize();
+                } else {
+                    this.win.unmaximize();
+                }
+            }
+
+            closeWindow (event){
+                this.win.close();
```

Comment:
Does the confirmation dialog still popup if downloads are running?
See: [Pre-Flight Check](https://github.com/manga-download/hakuneko/issues/955) Step 40

Reply:
Yup ! (added in test coverage)